pytest.ini

```diff
@@ -2,5 +2,5 @@
 addopts =
     -vv --durations=5
     -k-deselect
-    --benchmark-skip
+     # --benchmark-skip
```

Comment:
what do you think, should we keep the benchmarks by default? they seem to take in the order of 5sec all combined ... a bit annoying but still fine...

Reply:
Yes I think it should be kept. I would say it is important to test if some changes did not affect drastically the computation time.